shardy/dialect/sdy/ir/attrs.td

```diff
@@ -37,7 +42,7 @@ def Sdy_MeshAxis : AttrDef<Sdy_Dialect, "MeshAxis"> {
   let summary = "Named axis in a mesh";
   let parameters = (ins
       StringRefParameter<"name">:$name,
-      "int64_t":$size
+      AttrOrTypeParameter<"int64_t", "Size of this axis">:$size
```

Comment:
Should we use captial letter? Does it make sense in the generated doc? Lets be consistent across the file as I see non capital letters.

Reply:
I went through all parameters and used lowecase letters for all - you are right, it does make it more consistent.